### PR TITLE
Integration tests for istioctl managing webhooks

### PR DESCRIPTION
**Pull request by lei-tang**
Please provide a description for what this PR is for: https://github.com/istio/istio/issues/17061.

Design documents:
[1]https://docs.google.com/document/d/1VHyBre8943USOx_YEVtA18KfEoGTmLT1iBHESO5bzcA/edit#heading=h.qex63c29z2to

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[X] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: tests/integration/istioctl/webhook/webhook_test.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+			// Test that webhook statuses returned by running istioctl are as expected.
+			t.Log("check that webhook statuses returned by running istioctl are as expected ...")
+			args = []string{"experimental", "post-install", "webhook", "status"}
+			istioCtl = istioctl.NewOrFail(t, ctx, istioctl.Config{})
+			output, fErr = istioCtl.Invoke(args)
+			if fErr != nil {
+				t.Fatalf("error returned for 'istioctl %s': %v", strings.Join(args, " "), fErr)
+			}
+
+			fmt.Printf("output is %v\n", output)
```
Comment (nmittler) [inline tests/integration/istioctl/webhook/webhook_test.go]: remove

File: tests/integration/istioctl/webhook/webhook_test.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+// TestWebhookManagement tests "istioctl experimental post-install webhook" command.
+func TestWebhookManagement(t *testing.T) {
+	framework.
+		NewTest(t).
+		Run(func(ctx framework.TestContext) {
+			g := galley.NewOrFail(t, ctx, galley.Config{})
+			_ = pilot.NewOrFail(t, ctx, pilot.Config{Galley: g})
+
+			// Test that webhook configurations are enabled through istioctl successfully.
+			t.Log("check that webhook configurations are enabled through istioctl successfully ...")
```
Comment (nmittler) [inline tests/integration/istioctl/webhook/webhook_test.go]: is this log useful?

File: tests/integration/istioctl/webhook/webhook_test.go [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+			}
+			for _, regexp := range expectedRegexps {
+				if !regexp.MatchString(output) {
+					t.Fatalf("output didn't match for 'istioctl %s'\n got %v\nwant: %v",
+						strings.Join(args, " "), output, regexp)
+				}
+			}
+
+			// Test that webhook statuses returned by running istioctl are as expected.
+			t.Log("check that webhook statuses returned by running istioctl are as expected ...")
```
Comment (nmittler) [inline tests/integration/istioctl/webhook/webhook_test.go]: same question

File: tests/integration/istioctl/webhook/webhook_test.go (hunk 1 above)
Comment (lei-tang) [reply tests/integration/istioctl/webhook/webhook_test.go]: Done.

File: tests/integration/istioctl/webhook/webhook_test.go (hunk 2 above)
Comment (lei-tang) [reply tests/integration/istioctl/webhook/webhook_test.go]: Removed.

File: tests/integration/istioctl/webhook/webhook_test.go (hunk 3 above)
Comment (lei-tang) [reply tests/integration/istioctl/webhook/webhook_test.go]: Removed.

File: tests/integration/istioctl/webhook/webhook_test.go [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+	cfg.ValuesFile = "test-values/values-istio-dns-cert.yaml"
+	cfg.Values["global.operatorManageWebhooks"] = "true"
+}
+
+// TestWebhookManagement tests "istioctl experimental post-install webhook" command.
+func TestWebhookManagement(t *testing.T) {
+	framework.
+		NewTest(t).
+		Run(func(ctx framework.TestContext) {
+			g := galley.NewOrFail(t, ctx, galley.Config{})
```
Comment (nmittler) [inline tests/integration/istioctl/webhook/webhook_test.go]: You're already doing this in your main ... you don't need this here.

Are you even using these variables? If not you don't need this logic here or above.

Comment (lei-tang) [reply tests/integration/istioctl/webhook/webhook_test.go]: Removed.

**Pull request merged**